src/utils/convertAmount.js

```diff
@@ -0,0 +1,5 @@
+const convertAmount = amount => {
```

Comment:
Pls rename, be more explicit about what is the amount converted to.

Reply:
Something like, convertToLevels or similar.

Reply:
Yup, I agree